environments/local-docker.yaml

```diff
@@ -144,6 +144,9 @@ isapi:
   allowInvalidCertificate: true
   dataProtection:
     applicationName: "ISAPI Development"
+  serviceDependencies:
+    email:
+      subjectPrefix: dev
```

Comment:
Is there a prefix value set for test somewhere?

Reply:
at the moment, that will be in the static files.